### PR TITLE
Blog: v20.4.0 release post

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs/node/pull/48643

### CODE REVIEW

File: pages/en/blog/release/v20.4.0.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+Windows 32-bit Binary: https://nodejs.org/dist/v20.4.0/win-x86/node.exe \
+Windows 64-bit Binary: https://nodejs.org/dist/v20.4.0/win-x64/node.exe \
+Windows ARM 64-bit Binary: https://nodejs.org/dist/v20.4.0/win-arm64/node.exe \
+macOS 64-bit Installer: https://nodejs.org/dist/v20.4.0/node-v20.4.0.pkg \
+macOS Apple Silicon 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-darwin-arm64.tar.gz \
+macOS Intel 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-darwin-x64.tar.gz \
+Linux 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-linux-x64.tar.xz \
+Linux PPC LE 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-linux-ppc64le.tar.xz \
+Linux s390x 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-linux-s390x.tar.xz \
+AIX 64-bit Binary: *Coming soon* \
```
Comment (richardlau) [inline pages/en/blog/release/v20.4.0.md]: 🤔

Comment (richardlau) [reply pages/en/blog/release/v20.4.0.md]:
```suggestion
AIX 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-aix-ppc64.tar.gz \
```

Comment (RafaelGSS) [reply pages/en/blog/release/v20.4.0.md]: yep:

```
➜  nodejs.org (blog-v20.4.0) node ./scripts/release-post/index.mjs 20.4.0                                                                                                                                                                                   ✱
 "AIX 64-bit Binary" is Coming soon...
Release post created: /home/rafaelgss/repos/os/nodejs.org/pages/en/blog/release/v20.4.0.md
```

Comment (RafaelGSS) [reply pages/en/blog/release/v20.4.0.md]: Should I do something?

**Review by richardlau — APPROVED** [review]

File: pages/en/blog/release/v20.4.0.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+ARMv7 32-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-linux-armv7l.tar.xz \
+ARMv8 64-bit Binary: https://nodejs.org/dist/v20.4.0/node-v20.4.0-linux-arm64.tar.xz \
+Source Code: https://nodejs.org/dist/v20.4.0/node-v20.4.0.tar.gz \
+Other release files: https://nodejs.org/dist/v20.4.0/ \
+Documentation: https://nodejs.org/docs/v20.4.0/api/
+
+### SHASUMS
+
+```
+[INSERT SHASUMS HERE]
```
Comment (richardlau) [inline pages/en/blog/release/v20.4.0.md]: This is missing.

**Review by richardlau — CHANGES_REQUESTED** [review]
SHASUMS are missing

**Review by richardlau — APPROVED** [review]

**Pull request merged**